src/Toolkit/src/ComponentRepository/ComponentIdentity.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Ux\Toolkit\ComponentRepository;
+
+/**
+ * @author Jean-François Lépine
+ *
+ * @internal
+ */
+readonly final class ComponentIdentity
+{
+    public function __construct(
+        private string $vendor,
+        private ?string $package = null,
+        private ?string $name = null,
```

Comment:
```suggestion
        private string $package = null,
        private string $name = null,
```

This is something i'll need to discuss with you guys